src/main/java/org/fundation/coding/adrian/katas/cyphers/charSorter/CharSorter.java

```diff
@@ -0,0 +1,42 @@
+package org.fundation.coding.adrian.katas.cyphers.charSorter;
+
+/**
+ * .
+ */
+public class CharSorter {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)